src/docker/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  keycloak:
+    build: .
```

```suggestion
    image: keycloak/keycloak:24.0
```
Andreas hat die Firewall_Freigabe bekommen, wir können jetzt direkt pullen